<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptDialog</class>
 <widget class="QDialog" name="OptDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Связь</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="cb_modbus_packets">
        <property name="text">
         <string>Выводить пакеты Modbus в окно сообщений</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>9</number>
      </property>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Пауза между СБРОСОМ и началом программирования: (мсек)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_delay">
        <property name="maximum">
         <number>99999</number>
        </property>
        <property name="singleStep">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb1">
        <property name="text">
         <string>кнопка &quot;Стереть&quot;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb2">
        <property name="text">
         <string>кнопка &quot;Запись загрузчика&quot;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cb_modbus_packets</tabstop>
  <tabstop>sb_delay</tabstop>
  <tabstop>cb1</tabstop>
  <tabstop>cb2</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>OptDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>OptDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
